examples/wifi-echo/server/esp32/main/EchoDeviceCallbacks.cpp

```diff
@@ -68,7 +68,9 @@ void EchoDeviceCallbacks::DeviceEventCallback(const ChipDeviceEvent * event, int
                 // of the peripheral, the following code send it over the current Rendezvous session.
                 if (rendezvousDelegate != NULL)
                 {
-                    rendezvousDelegate->Send(ipAddrStr);
+                    IPAddress addr;
+                    IPAddress::FromString(ipAddrStr, addr);
```

Comment:
should we check bool return code for the FromString? Or are we relyiung that ntoa is always ok?